network/stellar-disbursement-platform/admin-guide/configuring-sdp.mdx

```diff
@@ -0,0 +1,187 @@
+---
+id: configuring-sdp
+title: Configuring the SDP
+description: Understand the configuration options available for the Stellar Disbursement Platform (SDP)
+keywords: [SDP, configuration]
+sidebar_position: 45
+---
+
+Stellar Disbursement Platform services can be configured using a set of configuration options that are passed to the command line or set as environment variables.
+Depending on how you're using and deploying the SDP, these configurations can be set in a ConfigMap in Kubernetes, as environment variables in a Docker container, passed in as command line arguments, etc.
+
+In this section we will discuss the different configuration options available for the SDP.
+
+**Notes:**
+* Configurations that are tagged with *[secret]* are sensitive and should be stored securely.
+* These configurations are valid for version 2.x of the SDP.
+
+## SDP Core Service
+
+For the most up-to-date configuration, you can run the following command in the [stellar-disbursement-platform-backend git repository](https://github.com/stellar/stellar-disbursement-platform-backend):
+
+<CodeExample>
+
+```bash
+./stellar-disbursement-platform serve --help
+```
+
+</CodeExample>
+
+### Operational Configuration
+Operational Configuration allows controlling metrics, logging, and other operational aspects of the SDP Core Service.
+* `PORT` - The port on which the SDP Core Service will listen for incoming HTTP requests. Default: 8000.
+* `LOG_LEVEL` - Determines the verbosity level of logs. Options: "TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL", or "PANIC". Default: "TRACE".
+* `METRICS_PORT` - The port on which the SDP Core Service will expose its metrics. Default: 8002.
+* `METRICS_TYPE` - The type of metrics to expose. Options: "PROMETHEUS". Default: "PROMETHEUS".
+* `CRASH_TRACKER_TYPE` - The crash tracker type to use. Options: "SENTRY", "DRY_RUN". Default: "DRY_RUN".
```

Comment:
do we want to mention what DRY_RUN means? eg maybe  put (logging only) in parenthesis or something to that effect?

Reply:
Addressed in e015ac3772fccc33777dbb49b4c5474f46f86ae5